servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -176,6 +176,21 @@ void light_compute(vec3 N, vec3 L, vec3 V, vec3 light_color, float attenuation,
 			diffuse_brdf_NL = lightScatter * viewScatter * energyFactor;
 			*/
 		}
+#elif defined(DIFFUSE_OREN_NAYAR)
+
+	{
+		// see http://mimosa-pudica.net/improved-oren-nayar.html
+		float LdotV = dot(L, V);
+
+		float s = LdotV - NdotL * NdotV;
+		float t = mix(1.0, max(NdotL, NdotV), step(0.0, s));
+
+		float sigma2 = roughness * roughness; // TODO: this needs checking
+		vec3 A = 1.0 + sigma2 * (-0.5 / (sigma2 + 0.33) + 0.17 * diffuse_light / (sigma2 + 0.13));
```

Comment:
I've changed `diffuse_color` to `diffuse_light` since diffuse_color is removed in 4.0, maybe this should be a `specular_light`? cc @clayjohn